uwsgi_configurations.ini

```diff
@@ -0,0 +1,12 @@
+[uwsgi]
+# https://uwsgi-docs.readthedocs.io/en/latest/Options.html
+
+wsgi-file = linkedevents/wsgi.py
+processes = 12
```

Comment:
Here's something I forgot to do. This should be parametrized from environment variable. Should be rather easy.
12 processes is not the best option everywhere.

Reply:
Right. Done in 796993b4fa3b29cec80350449322e569b1b7f26a and it'll set the processes to the correct number if `UWSGI_PROCESSES` env var is set.